src/pages/blog.tsx

```diff
@@ -33,9 +33,9 @@ const StyledBlogPage = styled.div`
     .posts {
         display: flex;
         flex-wrap: wrap;
-        justify-content: space-between;
+        justify-content: space-around;
```

Comment:
@samuelpath Many Thanks. Sorry, you have to revert this change i.e:
```suggestion
        justify-content: space-between;
```

other then this we are in a good to merge this.